### PR TITLE
Front login components

### PR DESCRIPTION
**Pull request by sachabinder**
First implementation of the frontend. In particular, the landing page (login)
The user can select between student or teacher.
* If student, he can fill a form with the room ID and his nickname
* If teacher, he can fill a form with his email and password

:warning: The link to register should be connect to the register form component, not done yet.
Finally, the code should may be lint and the implementation optimized.

### CODE REVIEW

File: front/src/components/login_forms.js [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+                </Grid>
+            </Paper>
+        </Grid>
+    )
+}
+
+export const TeacherLogin = () => {
+    const buttonStyle = {
+        margin: "8px 0"
+    }
```
Comment (StephaneGrabsia) [inline front/src/components/login_forms.js]: DRY! I think you should put this definition outside of the component ;)

File: front/src/components/login_forms.js [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+                        style={buttonStyle}
+                    />
+                    <FormControlLabel
+                        control={
+                            <Checkbox
+                                name="checkedB"
+                                color="primary"
+                            />
+                        }
+                        label="Se souvenir de moi"
```
Comment (StephaneGrabsia) [inline front/src/components/login_forms.js]: Does this button is necessary for the student? I mean, the id room will change everytime, and he may want to change its username too.

File: front/src/components/register_form.js [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+    }
+    const avatarStyle = {
+        backgroundColor: 'green'
+    }
+    return (
+        <Grid>
+            <Paper elevation={10} style={paperStyle}>
+                <Grid align='center'>
+                    <h2 style={headerStyle}>Créer un compte</h2>
+                    <Typography variant='caption'>Replir ce formulaire pour créer votre compte professeur !</Typography>
```
Comment (StephaneGrabsia) [inline front/src/components/register_form.js]: Typo, it's "remplir"

File: front/src/containers/index.js [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+    }}>
+      <Paper elevation={10} style={paperStyle}>
+        <Grid align="center">
+          <img src={Logo} style={imgStyle} />
+        </Grid>
+
+        <Tabs
+          value={value}
+          onChange={handleChange}
+          aria-label="disabled tabs example"
```
Comment (StephaneGrabsia) [inline front/src/containers/index.js]: You should change this label. But good job, I followed the same tuto ;)

**Review by StephaneGrabsia — DISMISSED** [review]
Good job! It's very beautiful !
CR done

**Review by StephaneGrabsia — DISMISSED** [review]
Thanks!

**Review by SullivanCastro — APPROVED** [review]
Excellent work !

**Pull request merged**